spring-cloud-gcp-data-firestore/src/test/java/com/google/cloud/spring/data/firestore/it/FirestoreIntegrationTests.java

```diff
@@ -273,6 +273,14 @@ public void writeReadDeleteTest() {
 		assertThat(this.firestoreTemplate.deleteAll(User.class).block()).isEqualTo(2);
 		assertThat(this.firestoreTemplate.findAll(User.class).collectList().block()).isEmpty();
 
+		alice.setUpdateTime(null);
```

Comment:
I think it would be easier for us to troubleshoot in case of failure if `deleteAllById()` had a dedicated test method. We do combine different scenarios for integration tests when it's hard to set up each test individually, but in this case the two new objects need to be set up anyway (since Alice object got deleted a few lines above).